KlockanAPI/src/KlockanAPI.Application/DTOs/City/CityDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace KlockanAPI.Application.DTOs.City;
+
+public class CityDto
```

Comment:
In the project we are currently utilizing DTO in some areas instead of Dto. Let's define one.